src/components/MenuItem/MenuItem.jsx

```diff
@@ -68,23 +94,44 @@ export class MenuItem extends React.Component {
             &pound;{total.toFixed(2)}
           </span>
         </p>
-        <select value={value} onChange={this.changeBread}>
-          {breads.map((bread, i) => {
-            const { name, price } = bread;
+        <div className="row">
+          <FormControl>
+            <InputLabel shrink>Bread</InputLabel>
+            <Select
+              native
+              value={value}
+              onChange={this.changeBread}
+            >
+              {breads.map((bread, i) => {
+                const { name, price } = bread;
 
-            return (
-              <option key={i} value={i}>
-                {name} (+&pound;{price.toFixed(2)})
-              </option>
-            );
-          })}
-        </select>
-        {canBeHeated && (
-          <label onChange={this.changeHot}>
-            <input type="checkbox" /> <small>Hot? (+<Price value={hotPrice} />)</small>
-          </label>
-        )}
-        <button type="button" onClick={this.action}>+ Add</button>
+                return (
+                  <option key={i} value={i}>
+                    {name} (+&pound;{price.toFixed(2)})
+                  </option>
+                );
+              })}
+            </Select>
+          </FormControl>
+          {canBeHeated && (
+            <FormControlLabel
+              control={(
+                <Switch
+                  onChange={this.changeHot}
+                />
+              )}
+              label={HotSwitch}
+            />
+          )}
+          <Button size="small" variant="contained" color="primary" onClick={this.handleClick}>+ Add</Button>
+          <OrderConfirmation
+            addOrderOwner={this.addOrderOwner}
+            sandwich={this.state}
```

Comment:
Might be worth selecting what you need from state rather than just passing everything through as a prop called `sandwich`. `modalOpen` is already passed for example